app/views/school_groups/_sub_nav.html.erb

```diff
@@ -0,0 +1,33 @@
+<nav class="navbar navbar-expand-lg navbar-custom fixed-top fixed-top-sub-nav sub-navbar">
+  <div class="container">
+    <div>
+    </div>
+    <div>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#subnavbarNavDropdown" aria-controls="subnavbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="subnavbarNavDropdown">
+        <ul class="navbar-nav">
+          <li class="nav-item dropdown mt-2">
+            <a class="nav-link dropdown-toggle" data-toggle="dropdown" id="manage_school" href="#" role="button" aria-haspopup="true" aria-expanded="false"><%= t('school_groups.sub_nav.dashboards') %></a>
```

Comment:
Can you change the ids and for the various divs here? Both menus have an id of manage_school, and both menus have an id of manage_school_menu. They should have unique names. E.g. "school-list-menu" and "manage-school-group". You'll need to update the cross-references in the aria attributes.